types/yeoman-assert/index.d.ts

```diff
@@ -0,0 +1,188 @@
+// Type definitions for yeoman-assert 3.1
+// Project: https://github.com/yeoman/yeoman-assert
+// Definitions by: Toilal <https://github.com/Toilal>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+// copy/paste of node 'assert' module definitions as export * doesn't work with module.
```

Comment:
I wish I could find a way to `export * from 'assert'`, but it seems it's not possible :(